### PR TITLE
gh-1749 Display fiat balances with decimals

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #1749

### CODE REVIEW

File: Multisig/Logic/Models/TokenBalance.swift [hunk 1]
```diff
@@ -76,7 +76,7 @@ extension TokenBalance {
     static func displayCurrency(from serverValue: String, code: String) -> String {
         let number = serverCurrencyFormatter.number(from: serverValue) ?? 0
         let formatter = TokenFormatter()
-        let amount = Int256(number.doubleValue) * 100
+        let amount = Int256(number.doubleValue * 100)
```
Comment (biafra23) [inline Multisig/Logic/Models/TokenBalance.swift:79]: Does this work for more or less than 2 minor units? 
There are currencies that have 0, 1 or 3 minor units of currency.

https://en.wikipedia.org/wiki/ISO_4217#Minor_units_of_currency

Comment (biafra23) [reply Multisig/Logic/Models/TokenBalance.swift:79]: BTW: We should never use floating point types for Fiat values. Always integer including minor units. But that is something for another PR :-)

Comment (DmitryBespalov) [reply Multisig/Logic/Models/TokenBalance.swift:79]: some currencies have 3 units

**Review by DmitryBespalov — APPROVED** [review]
I think it mergeable

File: Multisig/Logic/Models/TokenBalance.swift (hunk 1 above)
Comment (elgatovital) [reply Multisig/Logic/Models/TokenBalance.swift:79]: apparently there is no out of the box support for ISO4217 in ios. NumberFormatter returns 2 decimal digits even for codes that should have 3 or 4. In fiat list that we get from cgw that would be CLF and IQD
Issue opened: https://app.zenhub.com/workspaces/mobile-development-board-60cc78740f488d000e996335/issues/gnosis/safe-ios/1837

**Pull request merged**